internal/pkg/config/import_services.go

```diff
@@ -1,5 +1,16 @@
 package config
 
-type ImportedServices struct {
+import (
+	istionetv1alpha3 "istio.io/api/networking/v1alpha3"
+)
+
+type ImportedServiceSet struct {
 	Rules []Rules `yaml:"rules"`
 }
+
+type ImportedService struct {
+	AppName         string
+	ServiceHostname string
```

Comment:
Wouldn't be service name and namespace enough? I think even if the source cluster uses custom cluster domain, i.e. other than `cluster.local`, we do not care about that, because we will use east-west gateway's host name and SNI for routing. So to avoid splitting this field I suggest the following change:
```suggestion
	Name string
        Namespace string
```

Reply:
Yes that's true indeed, I hadn't considered that the local and remote clusters might be using different cluster domains so it doesn't make sense to import the full hostname.

If I understood correctly we will still need to build the expected internal cluster hostname when we create a ServiceEntry but we should be able to build this with the imported information + retrieving the local cluster domain. For example if we import remote service httpbin in the httpbin namespace we will have something like:

```
apiVersion: networking.istio.io/v1alpha3
kind: ServiceEntry
metadata:
  name: import-httpbin
  namespace: istio-system
spec:
  hosts:
  - httpbin.httpbin.svc.cluster.local
  ports:
  - number: 443
    name: https
    protocol: HTTPS
  endpoints:
  - address: remote.gateway.example.com
    ports:
      https: 15443
```